src/app/ssmt/explore-opportunities/explore-opportunities-form/operations-form/operations-form.component.html

```diff
@@ -107,62 +132,82 @@
+          <label for="baselineFuelCost" class="baseline-value-label">Fuel Cost</label>
+          <div class="text-muted text-center baseline-value-input">{{ ssmt.operatingCosts.fuelCost }} $/{{settings.steamEnergyMeasurement}}
+          </div>
         </div>
-      </div>
-      <div class="form-group">
-        <label for="modificationFuelCost">Modification Fuel Cost</label>
-        <div class="input-group">
-          <input name="modificationFuelCost" type="number" step="any" id="modificationFuelCost" class="form-control"
+      </form>
+      <form class="explore-opps-form">
+        <div class="form-group explore-opps-formgroup">
+          <label for="modificationFuelCost" class="baseline-value-label">Fuel Cost</label>
+          <div class="input-group">
+            <input name="modificationFuelCost" type="number" step="any" id="modificationFuelCost" class="form-control"
             (focus)="focusField('fuelCost')" (input)="save()"
             [(ngModel)]="ssmt.modifications[exploreModIndex].ssmt.operatingCosts.fuelCost" />
-          <span class="input-group-addon units">$/{{settings.steamEnergyMeasurement}}</span>
+            <span class="input-group-addon units">$/{{settings.steamEnergyMeasurement}}</span>
+          </div>
         </div>
-      </div>
-    </form>
+      </form>
   </li>
-  <li>
-    <div class="checkbox">
-      <label for="modifyMakeupWaterCost">
+</div>
+
+  <li class="explore-opps-item">
+    <div class="checkbox mt-2">
+      <label for="modifyMakeupWaterCost" class="steam-check-label">
         <input type="checkbox" id="modifyMakeupWaterCost" value="true" (focus)="focusField('makeUpWaterCost')"
           (change)="toggleMakeupWaterCost()" [(ngModel)]="showMakeupWaterCost" /> Modify Make-up Water Cost
       </label>
     </div>
-    <form *ngIf="showMakeupWaterCost">
-      <div class="form-group">
-        <label for="baselineMakeupWaterCost">Baseline Make-up Water Cost</label>
-        <div class="input-group">
-          <input name="baselineMakeupWaterCost" type="number" step="any" id="baselineMakeupWaterCost"
-            class="form-control" (focus)="focusField('makeUpWaterCost')" (input)="save()"
-            [(ngModel)]="ssmt.operatingCosts.makeUpWaterCost" [disabled]="true" />
-          <span class="input-group-addon units">$/{{settings.steamVolumeMeasurement}}</span>
-        </div>
+  </li>
+  <hr class="mt-0 mb-0">
+
+<div *ngIf="showMakeupWaterCost">
+  <li class="explore-opps-header" >
+    <span class="explore-opps-title text-muted">Baseline</span>
+    <span class="explore-opps-title text-muted">Modifications</span>
+  </li>
+
+  <li class="explore-opps-item mb-4">
+    <form class="explore-opps-form">
+      <div class="text-center">
+        <label for="baselineMakeupWaterCost" class="baseline-value-label">Make-up Water Cost</label>
+        <div class="text-muted text-center baseline-value-input">{{ ssmt.operatingCosts.makeUpWaterCost }} $/{{settings.steamEnergyMeasurement}}</div>
```

Comment:
Looks like the unit changed here from `steamVolumeMeasurement` to `steamEnergyMeasurement` be careful copy/pasting..